doc/source/conf.py

```diff
@@ -104,16 +104,16 @@
 # Enusre all our internal links work
 nitpicky = True
 exclude_patterns = [  # glob-style
-    # these are all included directly in dev/index.rst:
-    'dev/decisions.rst',
-    'dev/deprecations.rst',
-    'dev/distributing.rst',
-    'dev/github.rst',
-    'dev/licensing.rst',
-    'dev/modules.rst',
-    'dev/newfeatures.rst',
-    'dev/releasing.rst',
-    'dev/versioning.rst',
+    # these are all included directly in dev/core-dev/index.rst:
+    'dev/core-dev/decisions.rst',
```

Comment:
After renaming the files, removing the exclude patterns, modifying the include directives, e.g.
```
.. include:: decisions.rst.inc
```
and waiting for it to rebuild from scratch, I found `/dev/core-dev/index.html` to be empty. Had to revert.

Reply:
Saw your [PR](https://github.com/mdhaber/scipy/pull/1/files) so I tried again.

The problem was that when I attempted to change the filename extension, macOS  appended `.rst` onto it again and then automatically turned on "Hide filename extension" for that file. So my files were actually `.rst.inc.rst` but it only showed me `.rst.inc`, as though I had successfully changed the extension. 

No `./` because all the files are in the same directory as `index.rst`.